projects/packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -407,7 +408,7 @@ public function remote_provision( $request ) {
 		$secrets = ( new Secrets() )->generate( 'authorize', $user->ID );
 
 		$response = array(
-			'jp_version'   => JETPACK__VERSION,
+			'jp_version'   => Constants::get_constant( 'JETPACK__VERSION' ),
```

Comment:
Did we consider using `Jetpack_Options::get_option( 'version' )` here? AFAIK the difference is that multiple projects use `Jetpack_Options` but `JETPACK__VERSION` is only defined in the Jetpack plugin

_(`Jetpack_Options` does not solve the issue of having multiple plugins with different versions)_

Reply:
I could have checked this beforehand, but `Jetpack_Options::get_option( 'version' )` is only used in the Jetpack plugin so, in reality, it will not provide any better standalone plugin support but will potentially break existing compatibility since `Jetpack_Option` return versions like `11.1-a.6:1656089684`